internal/command/volumes/create.go

```diff
@@ -127,6 +133,12 @@ func runCreate(ctx context.Context) error {
 		snapshotID = fly.StringPointer(flag.GetString(ctx, "snapshot-id"))
 	}
 
+	fsType := flag.GetString(ctx, "fs-type")
+
+	if fsType != "ext4" && fsType != "raw" {
+		return fmt.Errorf("FSType '%s' is not one of ext4 or raw", fsType)
+	}
```

Comment:
I know this is picky, but can we follow the pattern of passing FSType only if `--fs-type` flag was specified in command line?

ala
```go
        var fsType *string
	if IsSpecified(ctx, "fs-type") {
		v = GetString(ctx, "vm-cpu-kind")
		if v != "ext4 && v != "raw" {
			return fmt.Errorf("--fs-type must be set to 'ext4' or 'raw, got: %s'", v)
		}
		fsType = &v
	}
```